### DISCUSSION TITLE
amqp-node/amqplib nodejs18 support

### DISCUSSION THREAD

**Discussion by CHAITANYAMALINENI** [community/community#53038] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hello, I am using amqplib latest version for one of my nodejs projects using nodejs 16 currently. My management wants us to migrate to Nodejs 18. But I see "amqplib" is not supporting Nodejs 18 yet. Can someone help me know when will be the Nodejs 18 support will be released out for the below amqplib library.
https://github.com/amqp-node/amqplib

**Comment by queenofcorgis** [comment]
If you're asking how to use the piece of software that you found in a repository on GitHub, the best way to contact the maintainers of that software is to:

- Check the README for instructions on how to operate the software and pointers to documentation or troubleshooting info.
- Check the [SUPPORT file](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-support-resources-to-your-project), if one exists, for instructions on how to best contact the maintainers for support.
- Check the [CONTRIBUTING guide](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors), if one exists. Sometimes if there isn't a SUPPORT file, the CONTRIBUTING guide will give instructions on how to contact the maintainers for support.

All of these documents can be found, if they exist, in the repository where you found the software itself.


Please note: You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
